optaplanner-docs/src/main/asciidoc/RepeatedPlanning/RepeatedPlanning-chapter.adoc

```diff
@@ -56,54 +86,68 @@ There are two ways to avoid that:
 
 If we handle overconstrained planning with nullable variables, the overload entities will be left unassigned:
 
-image::RepeatedPlanning/overconstrainedPlanning.png[align="center"]
+image::RepeatedPlanning/OverconstrainedPlanning/overconstrainedPlanning.png[align="center"]
```

Comment:
Doesn't this break the image?
This PR only changes 1 file and that png is not in that new directory.